test/fixtures/data/blockquotes/104.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  type: 'blockquote',
+  id: '104',
```

Comment:
Ideally, in the future, both the `type` and the `id` would be omitted from these files. The mocking module would insert these values automatically by convention. That would help make things just a bit DRYer.

Reply:
Update: this whole `json:api` mocking function has been extracted into [`jsonapilite`](https://npmjs.org/package/jsonapilite) (named after sqlite) and expanded a bit to support the needs of #14. That extraction occurs in the branch for #14, which is based off of this branch.